.github/workflows/tests.yml

```
@@ -372,6 +382,20 @@ jobs:
           gh cache delete "cache-${{ steps.get-date.outputs.date }}"
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Set lxd channel env variable
+        if: always()
```

Just to avoid cluttering the list of active steps from other jobs.
```suggestion
        if: ${{ always() && startsWith(matrix.test, 'ui ') }}
```